### DISCUSSION TITLE
Are the 4 rail sections planned?

### DISCUSSION THREAD

**Discussion by VAArtemchuk** [Opinionated-Blueprints/10-Books-Full-of-Rails#93] [General]
I've tried doing a city-block base with only the 2 rail sections and they seem to have a rather low throughput. Especially at the station stackers. I used to use your blueprints before 2.0 with 4 laners and they were dream-like good. Never had problems at the level I've reached. 

I'm using my own system of providers/requesters that turns requests off when station reaches a designated target and then turns it back on when it is low again to reduce overall traffic (a simple R-S latch). Providers are set to receive a total_inventory/train_capacity <= stacker size. This is my honest attempt to simulate LTN functionality and it MOSTLY works. There are some weird leftover items on one of the 50 current stations that neither of the players present can trace, but it's beside the point. Most of my stations accept up to 5 trains of 4 wagons (5 length in total). This setup with a dedicated parking lot results in a somewhat annoying jam at the parking's exit point and some of the more active providers. It used to be pretty easy to optimize with a 4 laner entry point. 

Are you planning to make the 4 laners again? If not, how do you reduce deadlocks in a 2 lane stacker system?

**Comment by Bocian-1** [comment]
With the wider turns in 2.0, 4 lane is no longer possible on this grid, and what limits throughput is the intersections and imsufficient spacing betweeen them. If you're having throughput issues, I'd suggest upgrading to grade-separated intersections
